.github/workflows/release.yml

```
@@ -0,0 +1,47 @@
+name: Publish package to NPM
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust:
+          - stable
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Setup Node.js environment
+      uses: actions/setup-node@v3.8.1
+    
+    - name: Setup Deno
+      uses: denoland/setup-deno@v1.1.2
+    
+    - name: NPM install with caching
+      run: npm install
```

```suggestion
      run: npm ci
```

(I always miss this one – `npm ci` uses `package-lock.json` to run a _much faster_ install)